### ISSUE TITLE
Resample() does not update dropout masks during policy training

### ISSUE THREAD

**Issue by Mi-Przystupa** [mcgillmrl/prob_mbrl#8]
The way the code is written, resample does not actually change the dropout masks used during training the policy. The only scenario it does happen is if the dropout mask and input shapes are misaligned. 

My proposal is something to the affect of flagging this when the model is in evaluation mode.
For example, in  update_noise(self): 
```
if not self.training:
      self.update_concrete_noise() #other whatever it's called
```

**Comment by juancamilog** [comment]
thanks for catching that!

**Comment by Mi-Przystupa** [comment]
your welcome! I think you added a fix with https://github.com/mcgillmrl/prob_mbrl/commit/97bbfc6e13cf813d0207187d2c505092f7c3688e

so guess it's fine to close this
